project/CelebornBuild.scala

```diff
@@ -334,14 +383,14 @@ object CelebornWorker {
     .settings (
       commonSettings,
       libraryDependencies ++= Seq(
-        "com.google.guava" % "guava" % guavaVersion,
-        "commons-io" % "commons-io" % commonsIoVersion,
-        "io.netty" % "netty-all" % nettyVersion,
-        "org.apache.logging.log4j" % "log4j-1.2-api" % log4j2Version,
-        "org.apache.logging.log4j" % "log4j-slf4j-impl" % log4j2Version,
-        "org.fusesource.leveldbjni" % "leveldbjni-all" % leveldbJniVersion,
-        "org.roaringbitmap" % "RoaringBitmap" % roaringBitmapVersion,
-        "org.mockito" %% "mockito-scala-scalatest" % scalatestMockitoVersion % "test",
+        Dependencies.guava,
+        Dependencies.commonsIo,
+        Dependencies.ioNetty,
+        Dependencies.log4j12Api,
```

Comment:
it's not clear to me the scope of each dep, and there is possible that a dep may have different scopes in different modules

Reply:
In the current implementation, we're using a naming pattern like this:

```
${dependency}${scope}
```

For instance:

```
`log4j12Api` -> "org.apache.logging.log4j" % "log4j-1.2-api" % log4j2Version 
`log4j12ApiTest` -> "org.apache.logging.log4j" % "log4j-1.2-api" % log4j2Version % "test"
```
Moreover, a dependency lacking a scope suffix designates the default runtime scope.

Do you have a better suggestion for the name rules?

Reply:
I like the latter one

Reply:
updated